FoodApp/Data/ApplicationDbContext.cs

```diff
@@ -14,5 +14,6 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         {
         }
         public DbSet<FoodApp.Models.Dish> Dish { get; set; }
+        public DbSet<FoodApp.Models.Restaurant> Restaurant { get; set; }
```

Comment:
Is it possible to use `DbSet<Dish/Restaurant>` instead? Seems unnecesserly long.

Reply:
Easier to read :^)

Reply:
No it's not. It's unneeded repetition.